src/transpiler.ts

```diff
@@ -171,6 +172,9 @@ function applyPasses(ast: AST, options: TranspilationOptions & PrintOptions): AS
   printPassName('Input', options);
   printAST(ast, options);
 
+  // Reject code that contains identifiers starting with certain patterns
+  ast = RejectPrefix.map(ast);
```

Comment:
The idea is that later we will not mangle user defined identifiers, that's why need to reject the prefixes. This is all so that we improve readability of generated code.
It needs to before the passes because the ast sanity checks would fail if there is a pass before the tuple fixes
Yea we don't need the assignment

Reply:
`TupleFixes` passes handles things such as: `(a, ) = (3, s[7][]);`. When the Sanity Checks are executed, they use `getNodeType` method which fails on this expression `(a, )` in `solc-typed-ast`. This means that adding `RejectPrefix` pass before `TupleFixes` will cause the Sanity Checks to fail. But still, `RejectPrefix` pass should be run first because `TupleFixes` already generates new variable names with the prefix `__warp_`

Reply:
How much work would it be to fix this in solc-typed-ast?